UI/DisplayLayoutScreen.cpp

```diff
@@ -68,50 +71,54 @@ bool DisplayLayoutScreen::touch(const TouchInput &touch) {
 	using namespace UI;
 
 	int mode = mode_->GetSelection();
-	if (g_Config.iSmallDisplayZoom == 0) { mode = -1; }
+	if (g_Config.iSmallDisplayZoomType == 2) { mode = -1; }
 
 	const Bounds &screen_bounds = screenManager()->getUIContext()->GetBounds();
-
 	if ((touch.flags & TOUCH_MOVE) && picked_ != 0) {
+		int touchX = touch.x - offsetTouchX;
+		int touchY = touch.y - offsetTouchY;
 		if (mode == 0) {
 			const Bounds &bounds = picked_->GetBounds();
 
 			int mintouchX = screen_bounds.w / 4;
```

Comment:
Nah minTouchX is not offset it sets bounds of the movement on the screen, I reused it in maxTouchX because it was same equation, ~ simplified, but that's not offset:).